### PR TITLE
(DOCSP-20940): Mention m1 support

### PR DESCRIPTION
**Pull request by jeff-allen-mongo**
## Staging

- [Release Notes](https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-20940/release-notes/#compass-1.29.4)
- [Install Docs](https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-20940/install/)

## JIRA

https://jira.mongodb.org/browse/DOCSP-20940

### CODE REVIEW

File: source/install.txt [hunk 1]
```diff
@@ -25,6 +24,21 @@ Select your operating system:
...
          - MongoDB 3.6 or later.
 
+         |compass-short| supports x64 and ARM64 architectures.
+         
+         M1 Support
+         ~~~~~~~~~~
+
+         You can run |compass| on M1 platforms.
+         
+         - If you run an operating system prior to macOS Big Sur, to run
```
Comment (davemungo) [inline source/install.txt:34]: Consider putting the newer operating system first, overtime fewer users will need to read both blurbs.

File: source/release-notes.txt [hunk 2]
```diff
@@ -375,6 +375,10 @@ New Features:
 
   - Node version 14
 
+Supported Platforms:
```
Comment (davemungo) [inline source/release-notes.txt:378]: Consider linking to the install considerations

**Review by davemungo — APPROVED** [review]
@jeff-allen-mongo thanks for the update. I left some optional considerations, but this LGTM

File: source/install.txt (hunk 1 above)
Comment (jeff-allen-mongo) [reply source/install.txt:34]: Updated.

File: source/release-notes.txt (hunk 2 above)
Comment (jeff-allen-mongo) [reply source/release-notes.txt:378]: Done.

**Pull request merged**